src/Test/Utilities/Portable/Assert/UseCultureAttribute.cs

```diff
@@ -51,16 +51,8 @@ public UseCultureAttribute(string culture)
         /// <param name="uiCulture">The name of the UI culture.</param>
         public UseCultureAttribute(string culture, string uiCulture)
         {
-#if NET472
             _culture = new Lazy<CultureInfo>(() => new CultureInfo(culture, useUserOverride: false));
             _uiCulture = new Lazy<CultureInfo>(() => new CultureInfo(uiCulture, useUserOverride: false));
-#elif NETCOREAPP2_1
-            _culture = new Lazy<CultureInfo>(() => new CultureInfo(culture));
```

Comment:
>)); [](start = 74, length = 3)

We used to have a distinction of `useUserOverride:` being true or false. Did something change that allows us to no longer care?

Reply:
The original code here preferred the `useUserOverride: false` approach. But when we went to netcoreapp1.1 originally that overload wasn't present and we were forced to pick a different one. Now with netcoreapp2.1 the overload is back and we can go with our original preferred approach.